src/cctag/ICCTag.hpp

```diff
@@ -41,6 +41,9 @@ class ICCTag
 
 	virtual ~ICCTag() {}
 
+	virtual ICCTag* clone() const = 0;
```

Comment:
It's because the detection returns a ptr_container which need clonable elements to be copiable.